src/DayPicker.js

```diff
@@ -421,17 +421,15 @@ export default class DayPicker extends Component {
     return React.cloneElement(navbarElement, props);
   }
   renderDayInMonth(day, month) {
-    let dayModifiers = [];
-    if (DateUtils.isSameDay(day, new Date())) {
-      dayModifiers.push(this.props.classNames.today);
+    const { today } = this.props.classNames;
+    const propModifiers = Helpers.getModifiersFromProps(this.props);
+    const dayModifiers = Helpers.getModifiersForDay(day, propModifiers);
+    if (DateUtils.isSameDay(day, new Date()) && !propModifiers.hasOwnProperty(today)) {
```

Comment:
Gotcha. Just about to ask how you guys wanted that handled syntacticly. I would normally use `in`, but there are a couple of reasons not to for performance reasons.